lib/apis/v2/ippool.go

```diff
@@ -37,25 +37,16 @@ type IPPool struct {
 type IPPoolSpec struct {
 	// The pool CIDR.
 	CIDR string `json:"cidr" validate:"omitempty,cidr"`
-	// Contains configuration for ipip tunneling for this pool. If not specified,
-	// then ipip tunneling is disabled for this pool.
-	IPIP *IPIPConfiguration `json:"ipip,omitempty"`
+	// Contains configuration for IPIP tunneling for this pool. If not specified,
```

Comment:
Since this is the source of truth for our APIs, should we add `This can be one of "Never", "Always" or "CrossSubnet".` in the comment?

Reply:
isn't that implied from the type?

Reply:
not CrossSubnet